lib/internal/Magento/Framework/Url/Decoder.php

```diff
@@ -24,14 +24,14 @@ public function __construct(
     }
 
     /**
-     * base64_decode() for URLs decoding
+     * Base64 decode for URL
      *
      * @param    string $url
      * @return   string
      */
     public function decode($url)
     {
-        $url = base64_decode(strtr($url, '-_,', '+/='));
+        $url = $url !== null ? base64_decode(strtr($url, '-_,', '+/=')) : '';
```

Comment:
I would prefer to fix that in
https://github.com/magento/magento2/blob/6bbf0aa7fcec36592caf28536d9ac47b220094ca/app/code/Magento/Widget/Controller/Adminhtml/Widget/Index.php#L53-L55

Reply:
Hello @xmav 
Thank you for your review.
We fixed it, please check